### ISSUE TITLE
Microsoft.CodeAnalysis.PerformanceSensitiveAnalyzers missing from nuget

### ISSUE THREAD

**Issue by chinwobble** [dotnet/roslyn-analyzers#2718]
#### Analyzer package

Microsoft.CodeAnalysis.PerformanceSensitiveAnalyzers

#### Package Version

Example: v2.9.4 (Latest)

I see in the repo that one of the packages that the build produces is `Microsoft.CodeAnalysis.PerformanceSensitiveAnalyzers` but I'm not able to see it in the readme.md at the root or the solution or on nuget. 
Can you advise what is the status of this package?

#### Expected behavior
Available for download on nuget
#### Actual behavior
Not available on nuget

**Comment by mavasani** [comment]
Tagging @genlu @sharwell - This is currently an internal-only package only meant for private consumption.

**Comment by chinwobble** [comment]
are there any plans to make this available on nuget?
It looks pretty useful.

**Comment by sharwell** [comment]
@chinwobble Yes, we want to make it public but we haven't don't any validation to make sure it meets quality or performance expectations.

**Comment by chinwobble** [comment]
@sharwell any idea when this will be available?

**Comment by genlu** [comment]
@chinwobble We are trying to dogfood it internally (see https://github.com/dotnet/roslyn/pull/41444), but don't have a date for final release yet. Meanwhile you can pick it up and try it from our myget feed:
https://dotnet.myget.org/feed/roslyn-analyzers/package/nuget/Microsoft.CodeAnalysis.PerformanceSensitiveAnalyzers

**Comment by KalleOlaviNiemitalo** [comment]
<https://github.com/dotnet/roslyn-analyzers/pull/2117> added _License notice for Roslyn Clr Heap Allocation Analyzer_ to  THIRD-PARTY-NOTICES.txt but not to assets/ThirdPartyNotices.rtf, which is included in the NuGet package. Based on the precedent in #2386, I guess the license notice should be added there too, before the package is published to nuget.org. Or replace assets/ThirdPartyNotices.rtf with THIRD-PARTY-NOTICES.txt in the NuGet packages.

**Comment by Youssef1313** [comment]
@genlu @sharwell What is remaining on this?
